docs/components/modeler/forms/form-element-library/forms-element-library-checklist.md

```diff
@@ -0,0 +1,36 @@
+---
+id: forms-element-library-checklist
+title: Checklist
+description: A form element to select multiple values from set options
+---
+
+A set of checkboxes providing data multi-selection for small datasets.
+
+![Form Checklist Symbol](/img/form-icons/form-checklist.svg)
+
+### Configurable properties
+
+- **Field label**: Label displayed on top of the checklist.
+- **Field description**: Description provided below the checklist.
+- **Key**: Binds the field to a form variable, see [data binding docs](../configuration/forms-config-data-binding.md).
+- **Options source**: Checklists can be configured with an options source defining the individual choices your user can make, see [options source docs](../configuration/forms-config-options.md).
+- **Disabled**: Disables the checklist, for use during development.
+
+### Datatypes
+
+Checklists can be bound to data of the `any[]` type, although for most practical cases we recommend `string[]` instead. The checklist component will correlate the bound data with the values of the different options.
```

Comment:
I've noticed a bit of differing voicing in this PR. For example, I would say "The checklist component ~~will~~ correlate**s**..."